functions.js

```diff
@@ -0,0 +1,107 @@
+//Task 1. Напишите функцию, которая принимает 2 числа и возвращает 1, если первое число больше, чем второе; 
+//-1, если первое число меньше, чем второе, и 0, если числа равны.
+//Попробуйте реализовать эту функцию не только с помощью if...else, 
+//но и с помощью тернарного оператора, и в виде стрелочной функции.
+//const num1 = +prompt('Enter a number');
+//const num2 = +prompt('Enter one more number');
+
+//function numberComparison(num1, num2) {
```

Comment:
if you already have read about arrow functions - use them

Reply:
From line 28 the arrow function in new commit